tests/units/test_commit.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+require File.dirname(__FILE__) + '/../test_helper'
+
+class TestCommit < Test::Unit::TestCase
+
+  def setup
+    set_file_paths
+  end
+
+  def test_add
```

Comment:
This should be in `tests/units/test_lib.rb` to ensure that things in `Git::Lib` are tested where they are expected.

The test name makes more sense as `#test_commit_with_date` in that file.